demos/linear_mixed_system/linear_mixed_system.py.rst

```diff
@@ -0,0 +1,328 @@
+Linear mixed fluid-structure system
+=================================================
+
+
+.. rst-class:: emphasis
+
+This tutorial is aimed to demonstrate the use of subdomain functionality and show how to describe a system consisting of multiple materials in Firedrake.
+
+The tutorial was contributed by `Tomasz Salwa <mailto:mmtjs@leeds.ac.uk>`__
+and `Onno Bokhove <mailto:O.Bokhove@leeds.ac.uk>`__.
+
+The work is based on the article "Variational modelling of wave-structure interactions with an
+offshore wind-turbine mast" by Tomasz Salwa, Onno Bokhove and Mark Kelmanson :cite:`Salwa:2017`. The authors gratefully acknowledge funding from European Commission, Marie Curie Actions - Initial Training Networks (ITN), project number 607596.
+
+The model considered consists of fluid with a free surface and an elastic solid. We will be using interchangeably notions of fluid/water and structure/solid/beam. For simplicity (and speed of computation) we consider a model in 2D, however it can be easily generalized to 3D. The starting point is the linearized version (domain is fixed) of the fully nonlinear variational principle. In non-dimensional units:
+
```

Comment:
Please use the more common UK spelling convention of -ise throughout (e.g. generalised).